packages/block-renderer/src/hooks/use-block-renderer-settings.ts

```diff
@@ -6,14 +6,14 @@ const useBlockRendererSettings = (
 	siteId: number | string,
 	stylesheet: string,
 	useInlineStyles = false,
-	queryOptions: UseQueryOptions< unknown, unknown, BlockRendererSettings > = {}
+	queryOptions: UseQueryOptions< unknown, Error, BlockRendererSettings > = {}
```

Comment:
We could eventually start using the `DefaultError` type once we adopt v5:

https://github.com/TanStack/query/blob/7c87c3d7b7fb0edf3a9ac69e2f29c0a3be07a669/packages/query-core/src/types.ts#L18C13-L18C25